### ISSUE TITLE
Access to data

### ISSUE THREAD

**Issue by VictorGFranco** [machinelearningsanfrancisco/San-Francisco-Traffic-Predictive-Model#2]
Contact the key people necessary to receive access.

**Comment by pringlized** [comment]
For Waze, a google account is required.  Then sign up for the [SDK program](https://developers.google.com/waze/intro-transport?refresh=1). 

Victor.. What do you think? Looks like they scrutinize signups for obvious reasons.  Feels like we need an initial signup with Organization and Main contact info.  Then each new member can sign up under our umbrella? 

Do you want to spear head joining the [Transport SDK](https://developers.google.com/waze/intro-transport?refresh=1) and be the main contact? Or would you like me to do that?  Thoughts?

**Comment by VictorGFranco** [comment]
Alright. I was going to do it but if you can take it of my plate it would be great. I'll leave it up to you.

I just finished an 8 hour Google Cloud Platform crash course that unifies everything under one roof including machine learning here at the Google San Francisco headquarters. I will spend the weekend synthesizing all this info and playing with it, which incorporates TensorFlow, since they gave me limited access and to the service.

Keep me in the loop.

Thanks

**Comment by pringlized** [comment]
Sounds good.  I assigned the task to myself.

**Comment by VictorGFranco** [comment]
Yeah. It would be one account and then everyone would have access to that account. Like an umbrella.
